superset-frontend/src/components/Card/index.tsx

```diff
@@ -17,23 +17,24 @@
  * under the License.
  */
 import React from 'react';
-import { styled } from '@superset-ui/core';
+import { SupersetTheme } from '@superset-ui/core';
```

Comment:
I don't like using the SupersetTheme directly like this... we can `useTheme` if you're set on the `css` method, but I want to maintain support for the `theme` provided by the `ThemeProvider` which effectively mutates/overrides the `SupersetTheme`

Reply:
Here we are not using the theme object. It's just the theme type. The values are coming from the `ThemeProvider`.
`css={(theme: SupersetTheme) => ({..})}`